src/octod/core.d

```diff
@@ -334,6 +334,55 @@ struct HTTPConnection
             ? MediaType.parse(accept) : MediaType.Default);
     }
 
+    /**
+        Sends PUT request to API server
+
+        Params:
+            url = GitHub API method URL (relative)
+            json = request body to send
+            accept = optional, request media type
+
+        Returns:
+            Json body of the response.
+     **/
+    Json put ( string url, Json json, MediaType accept )
```

Comment:
Putting this into `octod` is a bit confusing - I think whole connection helper should be moved to own separate module and octod package removed.